inst/include/dust/distr/binomial.hpp

```diff
@@ -166,34 +166,50 @@ inline HOSTDEVICE real_t btrs(rng_state_t<real_t>& rng_state, int n_int, real_t
   return draw;
 }
 
+template <typename real_t>
+HOSTDEVICE void rbinom_validate(int n, real_t p) {
```

Comment:
template the n type here instead of implicitly casting on L197 below?

Reply:
Or explicitly cast on L197

Reply:
Added a cast - can't be templated because of the %d in the sprintf expression